pkg/storaged/dialogx.jsx

```diff
@@ -177,18 +177,28 @@ const Validated = ({ errors, error_key, explanation, children }) => {
 
 const Row = ({ tag, title, errors, options, children }) => {
     if (tag) {
-        if (options.widest_title)
-            title = [ <div className="widest-title">{options.widest_title}</div>, <div>{title}</div> ];
-        return (
-            <tr>
-                <td className="top">{title}</td>
-                <td>
-                    <Validated errors={errors} error_key={tag} explanation={options.explanation}>
-                        { children }
-                    </Validated>
-                </td>
-            </tr>
+        let validated = (
+            <Validated errors={errors} error_key={tag} explanation={options.explanation}>
+                { children }
+            </Validated>
         );
+
+        if (title || title == "") {
+            if (options.widest_title)
+                title = [ <div className="widest-title">{options.widest_title}</div>, <div>{title}</div> ];
+            return (
+                <tr>
+                    <td className="top">{title}</td>
+                    <td>{validated}</td>
+                </tr>
+            );
+        } else {
+            return (
+                <tr>
+                    <td colSpan="2">{validated}</td>
+                </tr>
+            );
+        }
```

Comment:
This change and dropping the unused `Intermission` are unrelated, but no biggie.

Reply:
Intermission was a user of the (undocumented) special cases for the "title" attribute, so it was good to get it out of the way when changing those special cases.